qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -74,27 +90,15 @@
             pre_command = "echo 3 > /proc/sys/vm/drop_caches"
             pre_command_noncritical = yes
             variants:
-                - 2M_hugepage:
-                    # default pagesize is 2M, 2G guest memory need to allocate
-                    # in hugepage, so page nubmer is:
-                    #    target_hugepages = (size_mem / page_size) * 2 + 10
-                    target_hugepages = 2058
-                    expected_hugepage_size = 2048
-                - 16M_hugepage:
-                    only pseries
-                    target_hugepages = 266
-                    expected_hugepage_size = 16384
-                - 1G_hugepage:
-                    # Warning:
-                    #     Please don't forget to update host kernel line to enable
-                    # 1G hugepage support before lanuch this test, else test will
-                    # failed.
-                    target_hugepages = 5
-                    expected_hugepage_size = 1048576
-                    only one
```

Comment:
In previous tests,only several test cases were executed for 1G hugepage.The new patch will let all tests be executed for 1G hugepage.Is it essential ? Thanks.

Reply:
@MiriamDeng Not that essential, but the more the better for automation test. If you want to run less cases with 1G hugepage,  just narrow down the scope by specify the case names. e.g.  "hotplug_memory.after.guest_reboot.hotplug.backend_file.option_on.no_policy.one" will only generate 3 cases for the job.